Sources/WebDriver/Requests.swift

```diff
@@ -524,6 +524,55 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidtouchdoubleclick
+    public struct SessionTouchDoubleClick: Request {
+        public var session: String
+        public var element: String 
+
+        public var pathComponents: [String] { ["session", session, "touch", "doubleclick"] }
+        public var method: HTTPMethod {.post}
```

Comment:
Please review brace spacing:
```suggestion
        public var method: HTTPMethod { .post }
```